tests/src/test/scala/cats/tests/ChainSuite.scala

```diff
@@ -155,6 +181,34 @@ class ChainSuite extends CatsSuite {
     }
   }
 
+  test("fromOption should be consistent with one") {
+    forAll { (a: Int) =>
```

Comment:
I don't see how the value of `a` matters here. What about a non-property test where you use `()` as the value.

same comment for below.

Reply:
fixed